src/serializers/decoders/current/v16/decode_graph_entities.c

```
@@ -0,0 +1,496 @@
+
+		// introduce n to each relevant index
+		for (int i = 0; i < nodeLabelCount; i++) {
+			Schema *s = GraphContext_GetSchemaByID(gc, labels[i], SCHEMA_NODE);
+			ASSERT(s != NULL);
+
+			// index node
+			if(PENDING_IDX(s)) Index_IndexNode(PENDING_IDX(s), &n);
+		}
+	}
+
+	ASSERT(prev_graph_node_count + node_count == Graph_NodeCount(gc->g));
```

cppcheck

[error] Null pointer dereference

(nullPointer)